spec/integration/mixins/id_to_uri_spec.rb

```diff
@@ -24,12 +24,14 @@ class SomeTestClass
 
   it 'should throw BadRequest for -1' do
     expect do
-      SomeTestClass.identifier_to_uri(@opts, -1)
-    end.to raise_error(RestClient::BadRequest)
+      SomeTestClass.identifier_to_uri(@opts, '-1')
+    end.to raise_error(RestClient::BadRequest) { |error|
+      error.response.should match /does not match with value '-1'/
```

Comment:
Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.